python-flask/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: flask_app
+    environment:
+      FLASK_ENV: development
```

Comment:
I would change this environment for a .env and then call the environment file here instead.

Reply:
This we need to agree on. My other two Docker compose uses same syntax Simon. Small detail but we just need a standard imo.

Reply:
changed it to use the variables from the .env file